src/components/partnership/partnership.service.ts

```diff
@@ -734,4 +753,8 @@ export class PartnershipService {
       node('node', label, { active: true }),
     ]);
   }
+
+  protected canAddFundingType(types: PartnershipType[] | undefined): boolean {
+    return (types || []).includes(PartnershipType.Managing) ? true : false;
```

Comment:
This could be simplified
```suggestion
    return types?.includes(PartnershipType.Managing);
```

Reply:
null coalescing would be better too as it would skip the includes check all together.